internal/scheduler/job/repository_run.go

```diff
@@ -124,7 +125,7 @@ func (r *Repository) CompleteRun(ctx context.Context, runId string, nextRunIn ti
 	run.PrivateId = runId
 	_, err := r.writer.DoTx(ctx, db.StdRetryCnt, db.ExpBackoff{},
 		func(r db.Reader, w db.Writer) error {
-			rows, err := r.Query(ctx, completeRunQuery, []interface{}{runId})
+			rows, err := r.Query(ctx, completeRunQuery, []interface{}{completed, total, runId})
```

Comment:
Should this be using the reader and not the writer, given that the query is an update? Not too sure if it matters.